cluster-autoscaler/cloudprovider/kubemark/kubemark_other.go

```diff
@@ -34,30 +34,40 @@ const (
 	ProviderName = "kubemark"
 )
 
+// KubemarkCloudProvider implements CloudProvider interface.
 type KubemarkCloudProvider struct{}
 
+// BuildKubemarkCloudProvider builds a CloudProvider for kubemark. Builds
```

Comment:
Not sure these are the right comments, provided that most of the methods return an ErrNotImplemented.

Reply:
kubemark only works on linux environment, so this function in this file is Dummy implementation and returns ErrNotImplemented.

Reply:
It doesn't build outside of linux. That's why I think instead of copy-pasting the comments it's better to phrase all the comments sth like this:
//BuildKubemarkCloudProvider is not implemented for non-linux environments.

But now I see that there already are methods here with comments in the same way you added them, so I guess LGTM